packages/clay-layout/src/Col.tsx

```diff
@@ -0,0 +1,85 @@
+/**
+ * SPDX-FileCopyrightText: © 2020 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import classNames from 'classnames';
+import React from 'react';
+
+type TColSize = boolean | number | 'auto';
+
+interface IProps extends React.HTMLAttributes<HTMLDivElement> {
+	/**
+	 * Position in which to align column
+	 */
+	align?: 'start' | 'center' | 'end';
+
+	/**
+	 * The number of columns to span on extra-small devices
+	 */
+	xs?: TColSize;
+
+	/**
+	 * The number of columns to span on medium devices
+	 */
+	md?: TColSize;
+
+	/**
+	 * The number of columns to span on large devices
+	 */
+	lg?: TColSize;
+
+	/**
+	 * The number of columns to span on small devices
+	 */
+	sm?: TColSize;
+
+	/**
+	 * The order in which the column should be displayed
+	 */
+	order?: number | 'first' | 'last';
+
+	/**
+	 * The number of columns to span on all devices
+	 */
+	size?: TColSize;
+}
+
+const ClayCol: React.FunctionComponent<IProps> = ({
+	align,
+	children,
+	className,
+	lg,
+	md,
+	order,
+	size,
+	sm,
+	xs,
+	...otherProps
+}) => {
```

Comment:
Nice! 

We didn't add `align` and `order` to our JSP implementation... might be worth doing based on this! 🤔

Reply:
My goal was to mimic the docs examples as much as possible without having the user to have to add custom classes.

Reply:
We even don't recommend to use `order` since it causes accessibility problems of tab perception, I would not add it to avoid encouraging its use